big-int/src/bigint.hpp

```diff
@@ -31,123 +31,94 @@
 #include <iostream>
 #include <string>
 
-namespace libbig
-{
-class largeInt
-{
-  private:
-    bool sign;
-    std::string number;
-
-  public:
-    largeInt()
-    {
-    }
-
-    largeInt(int)
-    {
-    }
-    largeInt(int64_t)
-    {
-    }
-    largeInt(long long)
-    {
-    }
-    largeInt(std::string)
-    {
-    }
-
-    largeInt operator+(largeInt);
-    largeInt operator+(int);
-    largeInt operator+(int64_t);
-    largeInt operator+(long long);
-
-    largeInt operator-(largeInt);
-    largeInt operator-(int);
-    largeInt operator-(int64_t);
-    largeInt operator-(long long);
-
-    largeInt operator*(largeInt);
-    largeInt operator*(int);
-    largeInt operator*(int64_t);
-    largeInt operator*(long long);
-
-    largeInt operator/(largeInt);
-    largeInt operator/(int);
-    largeInt operator/(int64_t);
-    largeInt operator/(long long);
-
-    largeInt operator%(largeInt);
-    largeInt operator%(int);
-    largeInt operator%(int64_t);
-    largeInt operator%(long long);
-
-    largeInt &operator+=(largeInt);
-    largeInt &operator+=(int);
-    largeInt &operator+=(int64_t);
-    largeInt &operator+=(long long);
-
-    largeInt &operator-=(largeInt);
-    largeInt &operator-=(int);
-    largeInt &operator-=(int64_t);
-    largeInt &operator-=(long long);
-
-    largeInt &operator*=(largeInt);
-    largeInt &operator*=(int);
-    largeInt &operator*=(int64_t);
-    largeInt &operator*=(long long);
-
-    largeInt &operator/=(largeInt);
-    largeInt &operator/=(int);
-    largeInt &operator/=(int64_t);
-    largeInt &operator/=(long long);
-
-    largeInt operator%=(largeInt);
-    largeInt operator%=(int);
-    largeInt operator%=(int64_t);
-    largeInt operator%=(long long);
-
-    //   PREFIX OPERATORS
-
-    //   POSTFIX OPERATORS
-
-    void operator=(largeInt);
-    void operator=(int);
-    void operator=(int64_t);
-    void operator=(long long);
-
-    bool operator==(const largeInt &);
-    bool operator==(int);
-    bool operator==(int64_t);
-    bool operator==(long long);
-
-    bool operator!=(largeInt);
-    bool operator!=(int);
-    bool operator!=(int64_t);
-
-    bool operator<(largeInt);
-    bool operator<(int);
-    bool operator<(int64_t);
-    bool operator<(long long);
-
-    bool operator>(largeInt);
-    bool operator>(int);
-    bool operator>(int64_t);
-    bool operator>(long long);
-
-    bool operator<=(largeInt);
-    bool operator<=(int);
-    bool operator<=(int64_t);
-    bool operator<=(long long);
-
-    bool operator>=(largeInt);
-    bool operator>=(int);
-    bool operator>=(int64_t);
-    bool operator>=(long long);
-
-    friend std::istream &operator>>(std::istream &, largeInt &);
-    friend std::ostream &operator<<(std::ostream &, largeInt &);
-};
+namespace libbig {
+  class largeInt {
+    private:
+      bool sign;
+      std::string number;
+
+    public:
+      largeInt();
+
+      largeInt(int);
+      largeInt(long long int);
```

Comment:
Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285290)